libr/asm/arch/pyc/opcode.c

```diff
@@ -0,0 +1,287 @@
+#include "opcode.h"
+
+static version_opcode version_op[] = {
```

Comment:
can't we just specify the initial part of the version and use `strncmp` instead of `strcmp` to match the right opcode? So we don't need to specify every minor change in the version.

Reply:
That would be more complex than this. And I choose this to gain more attention from one who want to update for higher version python. He spend more time to figure out if the opcode's metainfo is accurate.